toponetx/utils/normalization.py

```diff
@@ -74,26 +73,25 @@ def compute_x_laplacian_normalized_matrix(L, Lx):
 
 
 def compute_kipf_adjacency_normalized_matrix(
-    A_opt, add_identity=False, identity_multiplier=1.0
-):
-    """
-    Normalize the adjacency matrix using Kipf's normalization.
+    A_opt: ndarray, add_identity: bool = False, identity_multiplier: float = 1.0
+) -> csr_matrix:
+    """Normalize the adjacency matrix using Kipf's normalization.
 
     Typically used to normalize adjacency matrices.
 
     Parameters
     ----------
-    A_opt : numpy.ndarray
+    A_opt : ndarray
```

Comment:
This should be reverted. Always use the full name in documentation (as per NumPy recommendation)